src/modules/fundingManager/bondingCurve/FM_BC_BondingSurface_Redeeming_v1.sol

```diff
@@ -81,7 +79,7 @@ contract FM_BC_BondingSurface_Redeeming_v1 is
     // Constants
 
     /// @dev Minimum collateral reserve
-    uint public constant MIN_RESERVE = 1 ether;
+    uint public MIN_RESERVE;
```

Comment:
Yeah thought about that too
Its just in the technicality a constant, at that It wont change anymore after we set it
I think I would keep it as its technically immutable, because we cant modify it without changing the code